packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -51,5 +37,6 @@ export type ProgressProps = Omit<ComponentProps<ProgressSlots>, 'size'> & {
 /**
  * State used in rendering Progress
  */
-export type ProgressState = ComponentState<ProgressSlots> &
-  Required<Pick<ProgressProps, 'indeterminate' | 'percentComplete' | 'thickness'>>;
+export type ProgressState = (ComponentState<ProgressSlots> & Required<Pick<ProgressProps, 'max' | 'thickness'>>) & {
+  value: ProgressProps['value'];
+};
```

Comment:
You can change this to be:

```suggestion
export type ProgressState = ComponentState<ProgressSlots> & Required<Pick<ProgressProps, 'max' | 'thickness'>> & Pick<ProgressProps, 'value'>;
```